custom_components/mqtt_vacuum_camera/utils/connection/decompress.py

```
@@ -0,0 +1,338 @@
+                                "%s: TIMING - Worker Hypfer processing: decompress=%.3fs, json=%.3fs",
+                                device_name, decompress_time, json_time
+                            )
+                        else:
+                            # Time the decompression step
+                            decompress_start_time = time.time()
+                            decompressed = await loop.run_in_executor(self._executor, rand256_decompress, payload)
+                            decompress_time = time.time() - decompress_start_time
+
+                            # Time the parsing step
+                            parse_start_time = time.time()
+                            parse_func = lambda: self._parser.parse_data(payload=decompressed, pixels=True)
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `parse_func` as a `def`

(E731)